src/org/loklak/LoklakServer.java

```diff
@@ -765,33 +765,23 @@ private static void checkServerPorts(int httpPort, int httpsPort) throws IOExcep
 
     	// check http port
         if(!httpsMode.equals(HttpsMode.ONLY)){
-	        ServerSocket ss = null;
-	        try {
-	            ss = new ServerSocket(httpPort);
+	        try(ServerSocket ss = new ServerSocket(httpPort)){
```

Comment:
Please make it like this -

```java
try (...) {
    ...
} catch (...) {
   ...
}
```
(see use of spaces)

It becomes more readable this was.